docs/specs/openapi.yaml

```
@@ -3,127 +3,139 @@ info:
           content:
             application/json:
               schema:
-                $ref: "#/components/schemas/PostLayersResponse"
+                $ref: "#/components/schemas/GetChangesResponse"
               example:
                 {
                   "type": "sync",
                   "status-code": 200,
                   "status": "OK",
-                  "result": true
+                  "result": [
+                    {
+                      "id": "4",
+                      "kind": "stop",
+                      "summary": "Stop service \"svc1\" and 1 more",
+                      "status": "Done",
+                      "tasks": [
+                        {
+                          "id": "7",
+                          "kind": "stop",
+                          "summary": "Stop service \"svc1\"",
+                          "status": "Done",
+                          "progress": {
+                            "label": "",
+                            "done": 1,
+                            "total": 1
+                          },
+                          "spawn-time": "2024-12-27T10:08:14.399194229+08:00",
+                          "ready-time": "2024-12-27T10:08:14.429319813+08:00"
+                        },
+                        {
+                          "id": "8",
+                          "kind": "stop",
+                          "summary": "Stop service \"svc2\"",
+                          "status": "Done",
+                          "progress": {
+                            "label": "",
+                            "done": 1,
+                            "total": 1
+                          },
+                          "spawn-time": "2024-12-27T10:08:14.399199354+08:00",
+                          "ready-time": "2024-12-27T10:08:14.432387271+08:00"
+                        }
+                      ],
+                      "ready": true,
+                      "spawn-time": "2024-12-27T10:08:14.399202521+08:00",
+                      "ready-time": "2024-12-27T10:08:14.432389313+08:00"
+                    }
+                  ]
                 }
-  /v1/services:
+  /v1/changes/{id}:
     get:
-      summary: List services
-      description: Fetch information about specific services (or all of them). Return results ordered by service name.
+      summary: Get a specific change
       tags:
-        - services
+        - changes and tasks
+      description: Fetch information about a Change given its ID.
       parameters:
-        - in: query
-          name: names
-          description: The names of the services to get. To get multiple services, specify this parameter multiple times. If not set, get all services.
+        - name: id
+          in: path
+          required: true
+          description: ID of the change.
           schema:
             type: string
       responses:
         "200":
-          description: List of services
+          description: Get change successfully.
```

For this one, I think it would be better to directly say what the response contains. How about:
```suggestion
          description: Information about the change.
```